src/createServer.js

```diff
@@ -1,3 +1,39 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+'use strict';
+
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const { isValidRequest } = require('./isValidRequest');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+    const text = normalizedURL.pathname.slice(1);
+    const toCase = normalizedURL.searchParams.get('toCase');
+
+    res.setHeader('Content-Type', 'application/json');
+
+    const statusText = isValidRequest(text, toCase);
+
+    if (statusText.errors.length) {
+      res.statusCode = 400;
+      res.end(JSON.stringify(statusText));
+
+      return;
+    }
+
+    const { originalCase, convertedText } = convertToCase(text, toCase);
+
```

Comment:
`You should respond with status 200 and OK status text.`

Check task requirements